### PR TITLE
Added a Splatoon squid

### PR DESCRIPTION
**Pull request by benmobbs**
May not work… May need to rejig creatures.js because there is no
delineator (,) in the list of creatures

### CODE REVIEW

**Comment by purge** [comment]
```
git fetch
git rebase origin/master
```

(probably worth using a visual tool to do the merge, or just remove the marks manually)

**Comment by benmobbs** [comment]
Cheers @purge , I seem to have wrapped my head around git and now I know the command line works as well - came bundled with XCode and needed to accept the T&Cs (of course..) I was using the OSX desktop app which now has far more useful things like a timeline making it easier to visualise the whole idea of comparing branches/forks and then pushing from master, pulling to master (I've got that right? Intuitively it sounds backwards, reminds me of camera aperture). It's how the cli, website and desktop all work together that takes a little grappling for a noob.

Admittedly my short term memory is so shit I can hardly remember what I've done which is why this is kind of platform is so useful for tracking. I knew what the problem was, it's actually actioning those changes via source control that I have little experience with - too much of a lone wolf doing the update/refresh dance.

tl;dr this is for my own understanding ;)

**Comment by benmobbs** [comment]
Said wrong things above. Noobs go here: http://rogerdudler.github.io/git-guide/
